src/librustdoc/formats/cache.rs

```diff
@@ -35,18 +35,18 @@ crate struct Cache {
     ///
     /// The values of the map are a list of implementations and documentation
     /// found on that implementation.
-    crate impls: FxHashMap<DefId, Vec<Impl>>,
+    crate impls: FxHashMap<RealDefId, Vec<Impl>>,
```

Comment:
Can't these be fake for blanket impls?

Reply:
```rust
pub trait A {}
pub trait B {}
impl<T: B> A for T {}
```

I don't know, but this documents just fine